zeppelin-web/yarn.lock

```diff
@@ -0,0 +1,4945 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Thanks for review @felixcheung 

> All yarn.lock files should be checked into source control (e.g. git or mercurial). This allows Yarn to install the same exact dependency tree across all machines, whether it be your coworker’s laptop or a CI server.

from https://yarnpkg.com/en/docs/yarn-lock